functions.php

```diff
@@ -512,47 +477,6 @@ function getWeekNumber($date, $connection2, $guid)
     }
 }
 
-/**
- * @deprecated in v23.
- */
-function getModuleEntry($address, $connection2, $guid)
```

Comment:
It seems that `getModuleEntry()` is still used in the latest version of [Reporting](https://gibbonedu.org/extend/#reporting) (v2.25). Do we wait for the module to update before removing the function?

Reply:
Reporting is a somewhat out of date module. The module developer will still have until v23 is released to update, should they choose, otherwise the current module is still (likely) compatible with v22 and previous versions.

Reply:
That's correct! I'll alert Andy Statham, the author, to this need.

Reply:
How do they manage their source code? It's not hard to patch it, but it's hard to submit any patch without a proper source hosting platform (e.g. GitHub, GitLab, BitBucket).

Reply:
They don't host online as far as I know. You could email a patch to andy@rapid36.com.

Reply:
Thanks :-)